youtube_dl/extractor/televizeseznam.py

```diff
@@ -0,0 +1,105 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import json
+import re
+
+from .common import InfoExtractor
+from ..utils import urljoin
+
+
+class TelevizeSeznamIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?televizeseznam\.cz/.+/(?P<display_id>.+)-(?P<id>[0-9]+)'
+
+    _GRAPHQL_URL = 'https://www.televizeseznam.cz/api/graphql'
+    _GRAPHQL_QUERY = '''query LoadEpisode($urlName : String){ episode(urlName: $urlName){ ...VideoDetailFragmentOnEpisode } }
+        fragment VideoDetailFragmentOnEpisode on Episode {
+            id
+            spl
+            urlName
+            name
+            perex
+        }
+'''
+
+    _TEST = {
+        'url': 'https://www.televizeseznam.cz/video/lajna/buh-57953890',
+        'md5': '40c41ade1464a390a0b447e333df4239',
+        'info_dict': {
+            'id': '57953890',
+            'display_id': 'buh',
+            'title': 'Bůh',
+            'description': 'Trenér Hrouzek je plný rozporů. Na pomoc si povolá i toho nejvyššího. Kdo to ale je? Pomůže mu vyřešit několik dilemat, která se mu v poslední době v životě nahromadila?',
+            'ext': 'mp4',
+        }
+    }
+
+    def extract_subtitles(self, spl_url, play_list):
+
+        if not play_list:
+            return None
+
+        subtitles = {}
+        for k, v in play_list.items():
+            if v.get('language'):
+                for ext in v.get('urls'):
+                    subtitles.setdefault(v['language'], []).append({
+                       'ext': ext,
+                       'url': urljoin(spl_url, v['urls'].get(ext))
+                    })
+        return subtitles
+
+    def _extract(self, ext, spl_url, play_list):
+        formats = []
+        for r, v in play_list.items():
+            format = {
+                'format_id': r,
+                'url': urljoin(spl_url, v.get('url')),
+                'protocol': 'https',
+                'ext': ext
+            }
+            if v.get('resolution'):
+                format.update({ 'width': v['resolution'][0], 'height': v['resolution'][1] })
```

Comment:
```suggestion
                format.update({'width': v['resolution'][0], 'height': v['resolution'][1]})
```
to fix flake8 errors

Reply:
thanks